TSC-Charter.md

```diff
@@ -55,6 +55,13 @@ TSC voting members are expected to regularly participate in TSC activities.
 
 A TSC voting member is automatically converted to a TSC regular member if
 they do not participate in three consecutive TSC votes.
+Regular membership is expected to be a temporary status, with the
```

Comment:
Can we give a specific duration?

Reply:
Maybe match the collaborator inactivity duration (I think was 9 months)